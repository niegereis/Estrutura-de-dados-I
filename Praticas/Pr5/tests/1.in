5
pedro 9.0 20.1.1111
guilherme 9.5 20.1.2222
maria 10.0 20.1.3333
jose 5.0 20.1.4444
clarice 7.0 20.1.5555
1 1