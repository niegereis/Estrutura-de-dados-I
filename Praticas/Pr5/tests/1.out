20.1.1111 9.0
20.1.2222 9.5
20.1.3333 10.0
20.1.4444 5.0
20.1.5555 7.0
